src/models/problem.ts

```diff
@@ -7,6 +7,11 @@ export const ProblemModel = types
     title: types.string,
     subtitle: "",
     sections: types.array(SectionModel)
+  })
+  .actions((self) => {
+    return {
+      getFullTitle: () => `${self.title}${self.subtitle ? `: ${self.subtitle}` : ""}`,
```

Comment:
- I like the idea of putting this here rather than relying on clients to do it. I should have thought of that when I wrote it originally. 😉 
- I would make it a get accessor, i.e. `get fullTitle: ...`
- I didn't know that backticks could be nested like that.